projects/igniteui-angular/src/lib/splitter/splitter.component.ts

```diff
@@ -349,8 +349,6 @@ export class IgxSplitterComponent implements AfterContentInit {
     }
 }
 
-export const SPLITTER_INTERACTION_KEYS = new Set('right down left up arrowright arrowdown arrowleft arrowup'.split(' '));
```

Comment:
PS: Could also keep the const (without exporting), IIRC just the `split` or any manipulation that bothered the tree-shaking, but something like this should be fine:
```ts
const SPLITTER_INTERACTION_KEYS = new Set(['right', 'down', 'left', 'up', 'arrowright', 'arrowdown', 'arrowleft', 'arrowup']);
```

Reply:
@damyanpetev This would be another way to resolve the issue, but I noticed that this logic is not used anywhere outside of the splitter component, which made me move it inside the component.